zerver/lib/actions.py

```diff
@@ -766,6 +766,32 @@ def do_scrub_realm(realm: Realm) -> None:
     RealmAuditLog.objects.create(realm=realm, event_time=timezone_now(),
                                  event_type=RealmAuditLog.REALM_SCRUBBED)
 
+def do_delete_user(user_profile: UserProfile) -> None:
+    do_deactivate_user(user_profile)
```

Comment:
Should this check if the user is already deactivated?

Reply:
do_deactivate_user handles that (does nothing if the user isn't active)